src/zen/websocket_server.cpp

```diff
@@ -227,6 +229,24 @@ class WsHandler
         rv->push_back(Pair("responsePayload", rspPayload));
         wsq.push(wse);
     }
+    
+    void sendTopQualityCertificates(const UniValue& mempoolCert, const UniValue& chainCert,
+                                    WsEvent::WsMsgType msgType, std::string clientRequestId = "")
+    {
+        // Send a message to the client:  type = eventType
+        WsEvent* wse = new WsEvent(msgType);
+        LogPrint("ws", "%s():%d - allocated %p\n", __func__, __LINE__, wse);
+        UniValue rspPayload(UniValue::VOBJ);
+        
+        rspPayload.push_back(Pair("mempool_top_quality_cert", mempoolCert));
```

Comment:
let's use CamelCase names as in other places:
`mempool_top_quality_cert` -> `mempoolTopQualityCert`
`chain_top_quality_cert` -> `chainTopQualityCert`